### PR TITLE
Do not fetch Twitter video URL while scraping

### LINKED ISSUE #1313: Do not request Twitter video URL while scraping
- Issue type: Enhancement

### Short Description
Parent issue: #1298.

Request to fetch Twitter video URL shouldn't be triggered while scraping the Tweets as it slows down the message collection rate. The requirement is to remove it from the main scraping process.

### PR DESCRIPTION
**Pull request by singhpratyush**
### Short description

Fixes #1313.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by singhpratyush** [comment]
The whole file appears as diff because line endings are changed to Unix style. I've added a comment for the changed part for easy review.

File: src/org/loklak/harvester/TwitterScraper.java [hunk 1]
```diff
@@ -1,858 +1,863 @@
...
+                images.add(image_url);
+                continue;
+            }
+            if ((p = input.indexOf("<source video-src")) >= 0 && input.indexOf("type=\"video/") > p) {
+                String video_url = new prop(input, p, "video-src").value;
+                videos.add(video_url);
+                continue;
+            }
+            if (input.indexOf("AdaptiveMedia-videoContainer") > 0) {
+                /* String tweetUrl = props.get("tweetstatusurl").value;
```
Comment (singhpratyush) [inline src/org/loklak/harvester/TwitterScraper.java]: Only this if loop's body is changed. I've commented out the process for getting the video URLs while commenting.

**Comment by singhpratyush** [comment]
@Achint08 @djmgit @hemantjadon @kavithaenair @SKrPl @vibhcool: Please review.

**Comment by vibhcool** [comment]
@singhpratyush , IMHO, removal of a feature isn't a solution to optimize a software.

Instead of removing the feature, we can add a service and move it to that service and keep video fetch as an option(something like `type=all_data`).

**Comment by singhpratyush** [comment]
@vibhcool: We've discussed this in #1298 and I'm following the same thing. These changes are just to avoid fetching URL while scraping.

Just to be clear, the methods are still there. In subsequent PRs, I'll be introducing the endpoint that was discussed in #1298 and later, we can add some field in Tweet object that tells that there ARE more videos and you may get them at `http://loklak.org/api/...json?id=http://twitter.com/status/.../...`.

**Comment by singhpratyush** [comment]
@hemantjadon: That would again take a day or two to get merged and I already have a PR waiting for this merge.

**Comment by vibhcool** [comment]
@singhpratyush , this diff body needs to be fixed. Could you fix it in this error here only?

**Pull request merged**